### ISSUE TITLE
Bug: tapping to change qubit doesn't work on mobile

### ISSUE THREAD

**Issue by bztravis** [Ecpii/bloch-m#5]
Tapping to change qubit doesn't work on mobile. 

iPhone 14 Pro Safari iOS 18.2

**Comment by Ecpii** [comment]
Yeah it seems to have weird behavior on Android as well - taps seem to be "buffered" in a sense. I've tried doing some debugging with adb and it seems like Chrome doesn't even receive the `pointerdown` event for a tap until the next tap:

https://github.com/user-attachments/assets/fa3c0b44-dd9a-4432-a95a-ee751c3f7fd1

Very perplexed by this and adb broke so can't debug this more now - leaving this to remember.

**Comment by Ecpii** [comment]
After a bit of testing, it seems like issues with mobile click events will be resolved in [this pull request on Tres.js](https://github.com/Tresjs/tres/pull/844). Tested on Android 15, Chrome 131.0.6778.200, as well as Chrome Dev Tools' touch emulation on Arc 131.0.6778.205.
